<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addShpFrm</class>
 <widget class="QWidget" name="addShpFrm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>加载矢量数据</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>22</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>数据名称：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OkBtn">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>108</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancleBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataName">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>20</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="selectDataPathBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择..</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataPath">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>56</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>58</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择路径：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
